src/components/DatePickerInline/types.ts

```diff
@@ -24,6 +24,7 @@ export interface BaseMonthProps {
     selectingMonth?: boolean;
     onPressDropdown?: (type: 'month' | 'year' | undefined) => void;
     onChange?: (value: number, type: 'month' | 'year') => void;
+    isDocked?: boolean;
```

Comment:
drop

Reply:
isDocked is required to handle some styles in Month component now, so dropping it is not a good idea. As I am using DatePickerInline as our Base now even for the DatePickerDocked and we do need to update some styles for Docked date picker. @shashank1010